src/cli/init.rs

```diff
@@ -184,6 +273,58 @@ mod tests {
     use std::path::{Path, PathBuf};
     use tempfile::tempdir;
 
+    #[test]
+    fn test_parse_conda_env_file() {
+        let example_conda_env_file = r#"
+        name: pixi_example_project
+        channels:
+          - conda-forge
+        dependencies:
+          - python
+          - pip:
+            - requests
+        "#;
+        let conda_env_file_data: CondaEnvFile =
+            serde_yaml::from_str(example_conda_env_file).unwrap();
+
+        assert_eq!(conda_env_file_data.name, "pixi_example_project");
+        assert_eq!(
+            conda_env_file_data.channels,
+            vec!["conda-forge".to_string()]
+        );
+
+        let (conda_deps, pip_deps) = parse_dependencies(conda_env_file_data.dependencies).unwrap();
+
+        println!("{conda_deps:?}");
+        assert_eq!(
+            conda_deps,
+            vec![
+                MatchSpec::from_str("python").unwrap(),
+                MatchSpec::from_str("pip").unwrap()
+            ]
+        );
+
+        assert_eq!(
+            conda_deps,
+            vec![
+                MatchSpec::from_str("python").unwrap(),
+                MatchSpec::from_str("pip").unwrap()
+            ]
+        );
+
+        assert_eq!(
```

Comment:
Nice! 
Could you add more dependencies? Including requirements for both conda and the pip dependencies.
Like one with `git+` like [this](https://github.com/google-research/google-research/blob/a11608cbc079aab875f354ada5253f8ebd4a5764/reset_free_learning/env.yml#L14) and one with an url or something like that. Which is okay if it just warns and uses non version or something else. 

And a one with a version and one with an extra. Also a good one would be to just do a big env.yml with all different sets and just use `assert_snapshot` from `insta` to get a less code intensive regression test.

Reply:
git dependencies would fail since they aren't supported by rip yet, should i still add them to the tests?